### PR TITLE
Issue #28: Fixing add_mappings function

### PR DESCRIPTION
**Pull request by bdrillard**
Addresses #28. Fixes the `add_mappings` function and adds test cases.

### CODE REVIEW

**Comment by ntlanglois** [comment]
+1 Looks good to me.

File: python/tests/test_bunsen_stu3.py [hunk 1]
```diff
@@ -330,15 +365,3 @@ def test_isa_custom(spark_session, bundles):
...
-def test_valueset_from_bundle(spark_session):
-  bundles = load_from_directory(spark_session, 'tests/resources/bundles/json', 1)
-
-  vs = extract_entry(spark_session, bundles, 'ValueSet')
-
-  value_sets = create_value_sets(spark_session) \
-    .with_value_sets(vs)
-
-  assert value_sets.get_values("http://hl7.org/fhir/ValueSet/example-extensional", "20150622") \
-           .count() == 4
```
Comment (rbrush) [inline python/tests/test_bunsen_stu3.py]: Was this test deleted for a reason, or was this a merge error?

**Comment by rbrush** [comment]
Looks good overall, everything is working on a local build of this change. Just had one question that I posted in the code.

File: python/tests/test_bunsen_stu3.py (hunk 1 above)
Comment (bdrillard) [reply python/tests/test_bunsen_stu3.py]: Good catch. That's definitely should not have been removed. Restored, https://github.com/cerner/bunsen/pull/29/files#diff-6af27a18039c37d475e74904160164f5L342

**Comment by rbrush** [comment]
Merged! Thanks for fixing this!

**Pull request merged**